src/serac/physics/solid_functional.hpp

```diff
@@ -190,26 +181,7 @@ class SolidFunctional<order, dim, Parameters<parameter_space...>, std::integer_s
   }
 
   /// @brief Destroy the Solid Functional object
-  ~SolidFunctional()
-  {
-    // Update the mesh with the new deformed nodes if requested
-    if (keep_deformation_ == FinalMeshOption::Deformed) {
-      *reference_nodes_ += displacement_.gridFunction();
-    }
-
-    // Build a new grid function to store the mesh nodes post-destruction
-    // NOTE: MFEM will manage the memory of these objects
-
-    auto mesh_fe_coll  = new mfem::H1_FECollection(order_, mesh_.Dimension());
-    auto mesh_fe_space = new mfem::ParFiniteElementSpace(displacement_.space(), &mesh_, mesh_fe_coll);
-    auto mesh_nodes    = new mfem::ParGridFunction(mesh_fe_space);
-    mesh_nodes->MakeOwner(mesh_fe_coll);
-
-    *mesh_nodes = *reference_nodes_;
-
-    // Set the mesh to the newly created nodes object and pass ownership
-    mesh_.NewNodes(*mesh_nodes, true);
-  }
+  ~SolidFunctional() {}
```

Comment:
The trivial destructor is an unexpected win. Even more evidence in favor of this change.